app/src/main/java/ch/hikemate/app/ui/map/RunHikeScreen.kt

```diff
@@ -156,6 +199,95 @@ fun RunHikeBottomSheet(
       scaffoldState = scaffoldState,
       sheetContainerColor = MaterialTheme.colorScheme.surface,
       sheetPeekHeight = MapScreen.BOTTOM_SHEET_SCAFFOLD_MID_HEIGHT,
-      modifier = Modifier.testTag(TEST_TAG_RUN_HIKE_SCREEN_BOTTOM_SHEET),
-      sheetContent = {}) {}
+      modifier = Modifier.testTag(TEST_TAG_BOTTOM_SHEET),
+      sheetContent = {
+        Column(
+            modifier = Modifier.padding(16.dp).weight(1f),
+        ) {
+          Text(
+              text = hikeRoute.route.name ?: stringResource(R.string.map_screen_hike_title_default),
+              style = MaterialTheme.typography.titleLarge,
+              textAlign = TextAlign.Left,
+              modifier = Modifier.testTag(TEST_TAG_HIKE_NAME))
+
+          // Elevation graph and the progress details below the graph
+          Column {
+            val hikeColor = Color(hikeRoute.route.getColor())
+            ElevationGraph(
+                elevations = elevationData,
+                styleProperties =
+                    ElevationGraphStyleProperties(
+                        strokeColor = hikeColor, fillColor = hikeColor.copy(0.1f)),
+                modifier =
+                    Modifier.fillMaxWidth()
+                        .height(60.dp)
+                        .padding(4.dp)
+                        .testTag(TEST_TAG_ELEVATION_GRAPH))
+
+            // Progress details below the graph
+            Row(
+                modifier = Modifier.fillMaxWidth().padding(bottom = 20.dp),
+                horizontalArrangement = Arrangement.SpaceBetween) {
+                  Text(
+                      text = stringResource(R.string.run_hike_screen_zero_distance_progress_value),
+                      style = MaterialTheme.typography.bodyLarge,
+                      fontWeight = FontWeight.Bold,
+                      textAlign = TextAlign.Left,
+                      modifier = Modifier.padding(top = 8.dp),
+                  )
+                  Text(
+                      text = "23% complete",
+                      style = MaterialTheme.typography.bodyLarge,
+                      color = hikeColor,
```

Comment:
Ahh, yes I thought about that too, but I thought to myself, that it would be more aesthetically unified and pleasing to have "everything" in the `hikeColor`. Don't you think the user will realise that this does not indicate an error? Because perhaps we could also consider re-thinking our color choices to not include red. If it really bothers you I can definitively hard-code it to green or some other color, but I think this solution with the `hikeColor`s is quite satisfying, since these colors also all work in dark & light mode.

Reply:
True, I personally like it as well. Maybe we can leave it as-is for now and discuss it for a future PR if we want to change it, but it should be discussed with the team and the stakeholders.